chemicaltoolbox/openbabel/ob_convert.xml

```diff
@@ -132,7 +132,11 @@
         #if float($ph) >= 0:
             -p $ph
         #end if
-
+        
+        #if $appendproperties != 'None':
```

Comment:
I think it can never be None, its not optional="true"

Reply:
It was none when I tried running without it - I can make it optional if that is preferred.

Reply:
The question is, should it be optional or not? If so I guess it does not hurt to add optional=true to the mix.

Reply:
it is optional